Mail/mimeDecode.php

```diff
@@ -621,7 +621,9 @@ function _quotedPrintableDecode($input)
         $input = preg_replace("/=\r?\n/", '', $input);
 
         // Replace encoded characters
-		$input = preg_replace('/=([a-f0-9]{2})/ie', "chr(hexdec('\\1'))", $input);
+        $input = preg_replace_callback('/=([a-f0-9]{2})/i', function ($matches) {
+            return chr(hexdec($matches[1]));
+        }, $input);
```

Comment:
I copied the old and new code to an isolated script and verified that they perform the same.